src/object/schedcontrol.c

```diff
@@ -72,14 +72,20 @@ static exception_t invokeSchedControl_Configure(sched_context_t *target, word_t
     }
 #endif /* ENABLE_SMP_SUPPORT */
 
-    if (target->scTcb && target->scRefillMax > 0) {
-        schedContext_resume(target);
-        if (SMP_TERNARY(core == CURRENT_CPU_INDEX(), true)) {
-            if (isRunnable(target->scTcb) && target->scTcb != NODE_STATE(ksCurThread)) {
-                possibleSwitchTo(target->scTcb);
+    assert(target->scRefillMax > 0);
```

Comment:
Love the asserts. They help verification work out what should be true and help us discovered during modification or before proofs if we're wrong.